Dockerfiles/ci/command.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+OPERATOR_DIR="${OPERATOR_DIR:-/project/operator_dir/}"
+OPERATOR_WORK_DIR="${OPERATOR_WORK_DIR:-/project/test_operator_work_dir/}"
+WORK_DIR="${WORK_DIR:-/project/output/}"
+
+# prepares operator metadata for a give operator bundle based on 
+# $OPERATOR_DIR environment variable
+prepare_operator_metadata () {
+    ansible-playbook -vvv -i localhost, --connection local operator-test-playbooks/prepare-operator-metadata.yml -e "operator_dir=$OPERATOR_DIR" -e "operator_work_dir=$OPERATOR_WORK_DIR" -e"work_dir=$WORK_DIR"
+    cp /project/operator-test-playbooks/parse_operator_metadata_results.json $WORK_DIR
+    test -f $WORK_DIR/parsed_operator_data.yml
+    test -f $WORK_DIR/parse_operator_metadata_results.json
+
+}
+
+
+test_for_report_failed_empty_alm_examples() {
+    echo "test_for_report_failed_empty_alm_examples: This test checks for failure when empty alm examples are found"
+    prepare_operator_metadata
+    grep -inr "Failed due to CSV.metadata.annotations\['alm-examples'\] is set to empty" $WORK_DIR
```

Comment:
We most not use recursive grep. 
We already know which file should contain the message. And we have to check that in the expected file and not anywhere.